datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -64,68 +65,89 @@ def items(self):
     def is_empty(self):
         return not self.items
 
-    def add_item(self, item: DatasetItem):
-        annotations = []
-        item_desc = {
-            "id": item.id,
-            "annotations": annotations,
-        }
-
-        if item.attributes:
-            item_desc["attr"] = item.attributes
-
-        if isinstance(item.media, Image):
+    @contextmanager
+    def save_media(self, item: DatasetItem):
```

Comment:
We usually use `save_media` for boolean variable to save or not. How about changing the name for avoiding confusions?

Reply:
Is it ok `context_save_media`?

Reply:
it looks good to me.

Reply:
Renamed it.